backend/cms/templatetags/page_filters.py

```diff
@@ -24,7 +24,7 @@ def page_translation_creator(page, language):
     page_translation = all_page_translations.filter(language__code=language.code)
     if page_translation.exists():
         return page_translation.first().creator
-    elif all_page_translations.exists():
+    if all_page_translations.exists():
```

Comment:
This is more than a code style change. Is this intended?

Reply:
Since there is a ´return´ in the if-block, it will never execute the code below if the first condition is true. And if the first condition is false, then it makes no difference whether we have ´elif´ or just ´if´. So it is indeed just a matter of code style, and our pylint config doesn't like unnecessary ´elif´ or ´else´ statements...